templates/02-counter/src/main.js

```diff
@@ -19,9 +19,42 @@ function updateUI() {
     Array.from(document.querySelectorAll('.sign-in')).map(it => it.style = "display: block;");
   } else {
     Array.from(document.querySelectorAll('.after-sign-in')).map(it => it.style = "display: block;");
+    contract.getCounter().then(count => {
+      document.querySelector('#show').classList.remove('loader')
+      document.querySelector('#show').innerText = count == undefined ? "calculating..." : count
+      document.querySelector('#left').classList.toggle('eye')
+      Array.from(document.querySelectorAll('button')).map(it => it.disabled = false)
```

Comment:
`.map` is for class when return value of function is used to construct new array. Otherwise just use `.forEach` (applies to other similar uses)